catalog/spatial/csw/spatial-csw-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -111,6 +111,17 @@
                             ref="inputTransformers"/>
     </reference-list>
 
+    <reference-list id="cswActionTransformers"
```

Comment:
Does it matter that CswActiontTransformerProvider does not implement CswActionTransformer?

Reply:
Yes, `CswActionTransformerProvider` is used to get an instance of the `CswActionTransformer` interface; it is not one itself